packages/patternfly-4/react-core/src/components/Button/Button.d.ts

```diff
@@ -17,7 +17,7 @@ export const ButtonType: {
 
 export interface ButtonProps extends HTMLProps<HTMLButtonElement> {
   children?: ReactNode;
-  component?: ReactType<ButtonProps>;
```

Comment:
This didn't work in my typescript app.  It had to be a string not a ReactType.  I suggest making this **component?: ReactType | string** if ReactType work for you.  I can't see how ReactType would work...  Do you have an example?

Reply:
i would be fine w/ `component?: ReactType | string;` or `component?: string;` as well. Both seem OK for me.

Reply:
Let's go with `component?: ReactType | string;` so we don't cause any breaking changes.